### PR TITLE
Try to consolidate guidance around AWS

### PR DESCRIPTION
**Pull request by benthorner**
Searching for 'AWS' returns quite a lot of content. This tries
to consolidate some of the docs to get people to the information
they need a bit faster.

### CODE REVIEW

File: source/manual/access-aws-console.html.md [hunk 1]
```diff
@@ -22,17 +22,23 @@ The (private) GitHub repo is [alphagov/gds-cli](https://github.com/alphagov/gds-
...
 
         gds aws govuk-integration-readonly -l
 
-If you're a PlatformHealthPowerUser and want to generate tokens for deploying Terraform in production, run:
+### Terraform
+
+👉 [Deploy AWS infrastructure with Terraform](/manual/deploying-terraform.html)
 
-        gds aws govuk-production-platformhealth-poweruser -e
+### Manual Hacks
```
Comment (issyl0) [inline source/manual/access-aws-console.html.md]: I dispute the term "hacks": they're not hacks, they're legit usages with the AWS CLI etc.

File: source/manual/concept-of-stacks.html.md [hunk 2]
```diff
@@ -1,44 +0,0 @@
----
```
Comment (issyl0) [inline source/manual/concept-of-stacks.html.md]: Stacks were a nice idea but ultimately didn't go any further than `blue` and they now just get in the way.

File: source/manual/access-aws-console.html.md (hunk 1 above)
Comment (benthorner) [reply source/manual/access-aws-console.html.md]: Sure. My comedy didn't quite come off - updated 👍

File: source/manual/concept-of-stacks.html.md (hunk 2 above)
Comment (benthorner) [reply source/manual/concept-of-stacks.html.md]: Interestingly, after putting this up I notice we've got 'green' DNS records defined on integration. But I still don't think it affects anything day-to-day.

**Pull request merged**